maintainers/backporting.md

```diff
@@ -0,0 +1,8 @@
+
+# Backporting
+
+Backports are handled by the backport action.
```

Comment:
Where do we find more information on this? Right now it's not clear at all how backports work.

Reply:
https://nixos.org/manual/nixpkgs/stable/#submitting-changes-stable-release-branches-automatic-backports

Reply:
```suggestion
To [automatically backport a pull request](https://github.com/NixOS/nix/blob/master/.github/workflows/backport.yml) to a release branch once it's merged, assign it a label of the form [`backport <branch>`](https://github.com/NixOS/nix/labels?q=backport).
```